nessie/customerRequests.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import utils.constants
+import json
+import re
+from models.customer import Customer
+from utils.exceptions import CustomerValidationError, NessieApiError, AddressValidationError
+
+
+class CustomerRequests:
+
+    def __init__(self, api_key):
+        self.key = api_key
+
+    # Methods Dealing With Customers
+
+    # Get customer for an Account
+    # Returns a Customer object who owns the AccountId
+    def get_customers_by_account_id(self, account_id):
+        if account_id is None:
```

Comment:
is there any additional validation we can do on account_id? e.g. does account_id only include certain characters, or is it always of some length?

Reply:
i think its a alphanumeric string of 23 characters but unsure if anything else would work